### ISSUE TITLE
Roster groups support

### ISSUE THREAD

**Issue by matwey** [ForNeVeR/kaiwa#76]
Hello, 

It would be great to support groups in roster, because I am used to specific order of my contacts.

**Comment by hi-q** [comment]
Sorting by status/alphabet in MUC user list also should be implemented

**Comment by Robbilie** [comment]
https://github.com/digicoop/kaiwa/pull/85

i added a pull request to the main branch, shouldnt be too hard to port, i couldnt rly decide wether to use this or the main repo for changes but since MUC was a bit buggy and for some reason didnt show my other rooms i chose the main one

**Comment by ForNeVeR** [comment]
@Robbilie first of all, big thanks for the contribution. I will port it to our fork.

Regarding your problem with MUC - could you please elaborate? Maybe create a separate issue?

**Comment by Robbilie** [comment]
public channels are not listed like in the normal kaiwa version, only the one i force join on login
also for some reason their names are not displayed but the JIDS

had this both in the normal and the React version

i guess i should stay on topic but some msgs in the react version appear grayed out even though they deliver, not sure what this means…

**Comment by vitalyster** [comment]
@Robbilie "normal kaiwa" shows _all_ groupchats from the "home" server, It's innacceptable for us (we have 1120 public groupchats right now) - but our version show bookmarked chatrooms (not sure kaiwa is able to bookmark rooms, but it works perfect with bookmarks stored from different clients). You see only room where you join -> you doesn't have chatroom bookmarks stored

**Comment by Robbilie** [comment]
@vitalyster makes total sense, will run into that issue at some point aswell, lets see if we can build a room browser into kaiwa then ;)

**Comment by ForNeVeR** [comment]
@vitalyster,

> not sure kaiwa is able to bookmark rooms

Yes it is able to. Just enter the room name into `add a room` box, and voila.
